src/test/skript/tests/syntaxes/effects/EffGlowingText.sk

```diff
@@ -0,0 +1,16 @@
+test "glowing sign blocks" when running minecraft "1.17.1":
+	set {_loc} to spawn of "world"
+	set block at {_loc} to sign
```

Comment:
The block needs to be set back to what it was. This test keeps the block as a sign potentially causing conflicts with other tests.

Reply:
isn't the block random anyways? so other tests can't rely on it being any specific type either way